### PR TITLE
Add redux-logger@3.0.6 w/ npm auto-update

### LINKED ISSUE #11031: [Request] Add redux-logger
**Library name:** redux-logger
**Git repository url:** https://github.com/evgenyrodionov/redux-logger
**npm package name or url** (if there is one): https://www.npmjs.com/package/redux-logger
**License (List them all if it's multiple):** MIT
**Official homepage:** 
**Wanna say something? Leave message here:** 

=====================

Notes from cdnjs maintainer:
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/.github/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/43928839-request-add-redux-logger?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #11031 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/evgenyrodionov/redux-logger
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/redux-logger
 * License and its reference: MIT, [ref](https://github.com/evgenyrodionov/redux-logger/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 33
   - Count of stars: 2096
   - Count of forks: 152
 * NPM download stats (optional):
   - Downloads in the last day: 35388
   - Downloads in the last week: 182405
   - Downloads in the last month: 763024

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #11031, cc @evgenyrodionov

### CODE REVIEW

**Comment by qwe19272375** [comment]
@cdnjs/intern3 please help me review this PR, thanks!

File: ajax/libs/redux-logger/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "keywords": [
+    "redux",
+    "logger",
+    "redux-logger",
+    "middleware"
+  ],
+  "author": {
+    "name": "Eugene Rodionov",
+    "url": "https://github.com/theaqua"
```
Comment (dakshshah96) [inline ajax/libs/redux-logger/package.json]: Let's stick to the author format in original [package.json](https://unpkg.com/redux-logger@3.0.6/package.json)?

`"author": "Eugene Rodionov (https://github.com/theaqua)"`

**Comment by qwe19272375** [comment]
Author field changed.
@cdnjs/intern3 please help me review it, thanks.

**Comment by qwe19272375** [comment]
@sufuf3 Do we need to separate the second commit into two individual one (add files because of structure and name)?

**Comment by sufuf3** [comment]
In my opinion, I think there is a detail description in detail commit message.
So, I think it's fine.

**Comment by qwe19272375** [comment]
@cdnjs/intern3 please help me review this PR, thanks.

**Comment by sufuf3** [comment]
`Add old versions of redux-logger, cc #11031` --> `Manually add old versions of redux-logger, cc #11031`
Let's let commit message clearly, and useful to trace git log.

**Comment by qwe19272375** [comment]
@sufuf3 the first commit line will exceed 50 characters if we change to `Manually add old versions of redux-logger, cc #11031`. Is it ok to do this?

**Comment by sufuf3** [comment]
If the last word can't separation, It's fine.

**Comment by qwe19272375** [comment]
Modified first line of the commit message.
@cdnjs/intern3 please help me review it, thanks.

**Comment by sufuf3** [comment]
The second detail commit message: `file name changed at v3.0.6`
I don't know what it means.
Could you give me more details?
Thank you.

**Comment by extend1994** [comment]
@sufuf3 I think it means naming scheme.

**Comment by sufuf3** [comment]
@qwe19272375 I don't think we need to manually add v0.0.1~v1.0.9. 
Please remove them.
Then, every thing LGTM.

Thank you.

**Comment by sashberd** [comment]
@sufuf3 @extend1994 @PeterDaveHello I can not rebase this PR to lates master. Please help me with this or close this PR

**Comment by sashberd** [comment]
Moved to #13712

**Pull request closed without merging**